web/client/plugins/FeedbackMask.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+const {connect} = require('react-redux');
+const {compose} = require('recompose');
+const {createSelector} = require('reselect');
+const {push} = require('react-router-redux');
+const {Button} = require('react-bootstrap');
+const {get} = require('lodash');
+
+const withMask = require('../components/misc/enhancers/withMask');
+const emptyState = require('../components/misc/enhancers/emptyState');
+const {isLoggedIn} = require('../selectors/security');
+const Message = require('../components/I18N/Message');
+const HTML = require('../components/I18N/HTML');
+
+const feedbackMaskSelector = createSelector([
+    state => get(state, 'feedbackMask', {}),
+    isLoggedIn,
+    state => !get(state, 'security')
+], ({loading, enabled, status, mode, errorMessage}, login, alwaysVisible) => ({
+    loading,
+    enabled,
+    status,
+    mode,
+    errorMessage,
+    login,
+    alwaysVisible,
+    showHomeButton: !alwaysVisible
+}));
+
+const HomeButton = connect(() => ({}), {
+    onClick: push.bind(null, '/')
+})(
+    ({onClick = () => {}}) => <Button
+        bsStyle="primary"
+        onClick={() => onClick()}>
+        <Message msgId="gohome"/>
+    </Button>
+);
+
+const MaskBody = emptyState(
```

Comment:
this component has no test

Reply:
@MV88 @mbarto @offtherailz This is not a reusable component but related to this plugin.
Where should I move this component to test it?

Reply:
components/buttons ?

Reply:
This is an error view like the typical 404 not found page, or access denied. 
We could reuse it in the future. For the moment is limited to 2 use cases (dashboards and maps), but it can be refactored quickly (not now, when another use case is required). 
@mbarto what do you think about `components/security` or a new `components/errors` ? 

About the name, instead of MaskBody, something like `ResourceUnavailable` ?

Reply:
components/errors is ok, and I agree with the ResourceUnavailable name